Projects and members pages design implementation.

juhath opened the pull request:
Added new designs to the projects and members pages. Also modified tests to cover the modified code.

juhath committed:
Projects and members pages design implementation.

Added new designs to the projects and members pages. Also modified tests to cover the modified code.

Ducky07 commented on styles/Profiles.module.scss:
```diff
@@ -0,0 +1,131 @@
+  }
+}
+
+// @media only screen and (max-width: 656.5px) {
+//   .profile {
+//     width: clamp(312px, 90%, 450px);
+//   }
+// }
```
Remove unused / commented out SCSS.

Ducky07 commented on styles/Profiles.module.scss:
```diff
@@ -0,0 +1,131 @@
+    align-self: start;
+  }
+
+  .text {
+    grid-column: 1 / span 2;
+    align-items: flex-start;
+  }
+
+  .socials {
+    grid-column: 1 / span 2;
+    align-items: flex-start;
+  }
+}
```
These classes use the same exact styles so they could be merged into one.

Ducky07 commented on styles/Profiles.module.scss:
```diff
@@ -0,0 +1,131 @@
+
+.name {
+  @extend %text-style;
+  font-size: 20px;
```
Use rem or em whenever possible with font-sizes

Ducky07 commented on styles/Repos.module.scss:
```diff
@@ -0,0 +1,68 @@
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
```
Same here, em or rem preferably.

Ducky07 reviewed (changes requested):
Looks good to me, good job.
Wait until @NinjaIndeed 's PR has gone through.

Here are couple things that you could still fine tune.

juhath committed:
Made the requested changes to Profiles.module.scss

Changed font-sizes to rem, removed unused code and merged two styles that had the same code.

juhath committed:
Made the requested changes to Repos.module.scss 

Changed the font-sizes to rem.

juhath committed:
Merge branch 'develop' into design-implementation
